ocs_ci/ocs/must_gather/must_gather.py

```diff
@@ -98,6 +100,27 @@ def validate_expected_files(self):
                     if "kind" not in f.read().lower():
                         self.files_content_issue.append(file)
 
+    def verify_ceph_file_content(self):
+        """
+        Verify ceph command does not return an error
+        https://bugzilla.redhat.com/show_bug.cgi?id=2014849
+        https://bugzilla.redhat.com/show_bug.cgi?id=2021427
+
+        """
+        if self.type_log != "CEPH" or self.ocs_version < version.VERSION_4_9:
+            return
+        pattern = re.compile("exit code [1-9]+")
+        for root, dirs, files in os.walk(self.root):
+            for file in files:
+                try:
+                    with open(os.path.join(root, file), "r") as f:
+                        data_file = f.read()
+                    exit_code_error = pattern.findall(data_file.lower())
+                    if len(exit_code_error) > 0 and "gather-debug" not in file:
+                        self.files_content_issue.append(os.path.join(root, file))
+                except Exception as e:
+                    logger.info(f"There is no option to read {file}, error: {e}")
```

Comment:
Should this be warning or error level log instead of info?

Reply:
done